app/src/main/res/values/strings.xml

```diff
@@ -12,5 +12,20 @@
     <string name="sign_up">Sign-Up</string>
     <string name="username">Username</string>
     <string name="textpassword">Password</string>
-    <string name="logout">LogOut</string>
+    <string name="logout">Logout</string>
+    <string name="app_logo_name">VRE</string>
+    <string name="app_title_name">Voice Recognition Emergency</string>
+    <string name="username_text">Email or Username</string>
+    <string name="password_text">Password</string>
+    <string name="forgot_password_text">Forgot your password?</string>
+    <string name="or_text">Or</string>
```

Comment:
```suggestion

```
Remove this and the one in the xml. Maybe we don't need to have an Or text

Reply:
Deleted the string name

Reply:
sounds good. You can go ahead and resolve conversation to close this comment.